crates/orchestrator/api.toml

```diff
@@ -23,6 +23,29 @@ This must be a POST request so we can update the OrchestratorState in the server
 received from the 'identity' endpoint
 """
 
+# POST the node's node index to generate public key for pubkey collection
+[route.postpubkey]
+PATH = ["pubkey/:node_index"]
+METHOD = "POST"
+":node_index" = "Integer"
+DOC = """
+Post a node's node_index so that its public key could be posted and collected by the orchestrator.
+"""
+
+# GET whether or not the config with all peers' public keys / configs are ready
+[route.peer_pubconfig_ready]
+PATH = ["peer_pub_ready"]
+DOC = """
+Get whether the node can collect the final config which includs all peer's public config/info like public keys, returns a boolean.
+"""
+
+# GET the updated config with all peers' public keys / configs
+[route.config_after_peer_collected]
```

Comment:
This looks like something that we could use for light client stuff as it contains `known_nodes_with_stake`. But the problem is, it doesn't contain the Schnorr verification keys.
However, Schnorr key shouldn't be in the consensus logic.

Before the stake table integration, if all keys are generated from the indexed seed, it might be good to generate the stake table in the same way.

Reply:
For Schnorr verification keys, is there any action item we want to take now for easier light client integration?
For stake table, do you mean we also want the stake be inputed as the same place as indexed seed?

Reply:
Not now. If we are not changing the way to generate keys before stake table integration, we are all good.